### DISCUSSION TITLE
brew install latexindent: OS unsupported

### DISCUSSION THREAD

**Discussion by tueda** [Homebrew/discussions#1027] [Linux] [answered]
I got an error message `OS unsupported` when I tried to install `latexindent` on Ubuntu 18.04/WSL (here is a [Gist log](https://gist.github.com/tueda/039aa95ffb74faeed4465bc153771590)). The same error occurred for the [linuxbrew/brew Docker image](https://hub.docker.com/r/linuxbrew/brew). But it's a Perl script, so I guess it should work.

Any ideas?

**Comment by Bo98** [comment, marked as answer]
Looks like this formula needs some fixes for it to work on Linux (notably not installing the `Mac::SystemDirectory` resource). You can open a linuxbrew-core issue or, if you can, open a homebrew-core PR with a fix.

**Reply by tueda** [reply]
Thank you for your comment. Then I will open an issue in `linuxbrew-core`. (I am not good at Perl, so making PR sounds difficult).

Should I mark your comment as an answer right now? Or after some fix is done?

**Reply by Bo98** [reply]
Up to you. I don't mind either way.

**Reply by tueda** [reply]
OK, I mark it as the answer, as I opened an issue: https://github.com/Homebrew/linuxbrew-core/issues/22542.
